src/electionguard_cli/cli_models/e2e_build_election_results.py

```diff
@@ -1,15 +1,20 @@
 from electionguard.election import CiphertextElectionContext
-from electionguard.manifest import InternalManifest
+from electionguard.manifest import InternalManifest, Manifest
 
 
 class BuildElectionResults:
     """The results of building an election, more specifically an internal manifest and context."""
 
     def __init__(
-        self, internal_manifest: InternalManifest, context: CiphertextElectionContext
+        self,
+        manifest: Manifest,
+        internal_manifest: InternalManifest,
+        context: CiphertextElectionContext,
     ):
+        self.manifest = manifest
         self.internal_manifest = internal_manifest
         self.context = context
 
+    manifest: Manifest
```

Comment:
The manifest object is already available in the election_inputs object (aka InputRetrievalStepBase).  Adding it here adds unnecessary complexity.